backend/tests/test_classify_tests.py

```
@@ -0,0 +1,494 @@
+        @pytest.mark.unit
+        @pytest.mark.integration
+        def test_two_marks():
+            assert True
+        """,
+    )
+    rc = classify_tests.check([classify_tests.scan_file(f)])
+    out = capsys.readouterr().out
+    assert rc == 1
+    assert "double-marked" in out
+    assert "test_two_marks" in out
+    assert "unit" in out and "integration" in out
```

Ruff (0.15.13)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)